packages/babel-parser/src/parser/statement.js

```diff
@@ -2005,12 +2011,8 @@ export default class StatementParser extends ExpressionParser {
       if (assertions) {
         node.assertions = assertions;
       }
-    } else {
-      if (expect) {
-        this.unexpected();
-      } else {
-        node.source = null;
```

Comment:
Actually, `node.source = null` is called in `maybeParseExportNamedSpecifiers` and `maybeParseExportDeclaration`. So we can remove this from here.